src/Middleware/SubstituteBindings.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+namespace LaravelDoctrine\ORM\Middleware;
+
+use Closure;
+use Doctrine\Common\Persistence\Proxy;
+use Illuminate\Contracts\Routing\Registrar;
+use Illuminate\Routing\Middleware\SubstituteBindings as IllumintateSubstituteBindings;
+use LaravelDoctrine\ORM\IlluminateRegistry;
+use ReflectionFunction;
+use ReflectionMethod;
+
+class SubstituteBindings extends IllumintateSubstituteBindings
```

Comment:
@HDsign this is confusing, I don't see any calls to `parent` other than the constructor. Is it really necessary to extend Eloquent's middleware?